staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/roundtrip_test.go

```diff
@@ -101,10 +102,6 @@ func TestRoundtrip(t *testing.T) {
 			name: "int64 zero",
 			obj:  int64(math.MinInt64),
 		},
-		{
```

Comment:
Added this case to the tracking issue. It's not roundtrippable with JSON, either, but it does fail differently. Related to the tracked TODO about int64 overflow behavior.